lib/sidekiq/worker.rb

```diff
@@ -355,13 +351,41 @@ def sidekiq_options(opts = {})
 
       def client_push(item) # :nodoc:
         raise ArgumentError, "Job payloads should contain no Symbols: #{item}" if item.any? { |k, v| k.is_a?(::Symbol) }
-        build_client.push_bulk(item.merge("args" => [item["args"]])).first
+        build_client.push(item)
```

Comment:
This reverts the change from #5182. Is it a conscious decision or omission?

Reply:
A mix. The code here is literally what `push` does, and the original code used `push`. I figured a quick "cleanup" to use the higher-order function made sense

Reply:
My intention was that `Sidekiq::Client#push` is no longer used in `Sidekiq::Worker` and `push_bulk` becomes the main interface. I left `push` because it's public and people depend on it.

I'm not sure if this makes sense or not. This is what I was thinking.

Reply:
Closing this PR in favor of the 7.0 PR (which doesn't yet have your code), but I would suggest that your refactor to have `push` rely on `push_bulk` is solid, but `push` remains an important and useful abstraction that should be used even within the Sidekiq codebase